scripts/deploy.sh

```diff
@@ -174,6 +185,42 @@ if [[ "${DEPLOY_OA}" == "yes" ]]; then
 
   cd ${OA_DIR}/playbooks/
 
+  #Distribute Magnum configuration files
+  if [[ "${DEPLOY_MAGNUM}" == "yes" ]]; then
+    cat > $OA_OVERRIDES <<'EOF'
+keystone_keystone_conf_overrides:
```

Comment:
That's a good question...we perhaps need a place to do this kind of logic where we're extending the OSA-provided files. I can't really think of a good, existing place for these kinds of changes at the moment.

Putting the vars in here requires that support personnel either have to edit the values here or do a deploy then edit the resulting files. Things like the `admin_project_domain_name` may vary across installations, though admittedly I'm not sure most of this would.